package.json

```diff
@@ -11,6 +11,7 @@
     "antd": "^4.18.3",
     "aws-sdk": "^2.1142.0",
     "axios": "^0.21.4",
+    "eslint": "^7.11.0",
```

Comment:
Did eslint needed to be add? If so did you check with jake or someone first?

Reply:
I am checking with Ash. It was required to get the program to run for me. I will fix it if needed.